utils/unittest/unittest.go

```diff
@@ -265,3 +267,11 @@ func Concurrently(n int, f func(int)) {
 	}
 	wg.Wait()
 }
+
+// AssertEqualBlocksLenAndOrder asserts that both a segment of blocks have the same len and blocks are in the same order
+func AssertEqualBlocksLenAndOrder(t *testing.T, expectedBlocks, actualSegmentBlocks []*flow.Block) {
```

Comment:
We can do this generally for any 2 lists of entities with:
```
assert.Equal(t, flow.GetIDs(expectedBlocks), flow.GetIDs(actualSegmentBlocks))
```

Reply:
https://github.com/onflow/flow-go/pull/1533/commits/382bd2b07c57196516ad4b817a74bd31659a1b92